web/src/main/java/com/futurewei/alcor/web/entity/securitygroup/SecurityGroupRule.java

```diff
@@ -15,7 +18,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.futurewei.alcor.common.entity.CustomerResource;
 
-public class SecurityGroupRuleEntity extends CustomerResource {
+public class SecurityGroupRule extends CustomerResource {
```

Comment:
Compared with Neutron SG rule, we need three more fields:

revision_number, created_at, updated_at.

Consider to add it into CustomerResource.

Reply:
ok

Reply:
Tracked by Issue https://github.com/futurewei-cloud/alcor/issues/246.